WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/BlazeCampaignStatusView.swift

```diff
@@ -54,39 +54,39 @@ struct BlazeCampaignStatusViewModel {
         switch status {
         case .created, .processing, .scheduled, .canceled:
             self.textColor = UIColor(
-                light: UIColor(fromHex: 0x4F3500),
-                dark: UIColor(fromHex: 0xDEB100)
+                light: .muriel(name: .yellow, .shade80),
+                dark: .muriel(name: .yellow, .shade10)
             )
             self.backgroundColor = UIColor(
-                light: UIColor(fromHex: 0xF5E6B3),
-                dark: UIColor(fromHex: 0x332200)
+                light: .muriel(name: .yellow, .shade5),
+                dark: .muriel(name: .yellow, .shade90)
             )
         case .rejected:
             self.textColor = UIColor(
-                light: UIColor(fromHex: 0x8A2424),
-                dark: UIColor(fromHex: 0xF86368)
+                light: .muriel(name: .red, .shade70),
+                dark: .muriel(name: .red, .shade10)
             )
             self.backgroundColor = UIColor(
-                light: UIColor(fromHex: 0xFACFD2),
-                dark: UIColor(fromHex: 0x451313)
+                light: .muriel(name: .red, .shade5),
+                dark: .muriel(name: .red, .shade90)
             )
         case .active, .approved:
             self.textColor = UIColor(
-                light: UIColor(fromHex: 0x00450C),
-                dark: UIColor(fromHex: 0x00BA37)
+                light: .muriel(name: .green, .shade80),
+                dark: .muriel(name: .green, .shade10)
             )
             self.backgroundColor = UIColor(
-                light: UIColor(fromHex: 0xB8E6BF),
-                dark: UIColor(fromHex: 0x003008)
+                light: .muriel(name: .green, .shade5),
+                dark: .muriel(name: .green, .shade90)
             )
         case .finished:
             self.textColor = UIColor(
-                light: UIColor(fromHex: 0x02395C),
-                dark: UIColor(fromHex: 0x399CE3)
+                light: .muriel(name: .blue, .shade80),
+                dark: .muriel(name: .blue, .shade10).lightVariant() /// Explicitly using the light variant of blue
```

Comment:
In ColorPalette.xcassets the Blue color has different shades for light vs dark. So unless we specify which variant we're using, it'll default to using the dark mode color defined in the assets file. I didn't want to delete the dark variant defined in the assets file because I wasn't sure how that would affect other areas.

Reply:
Oh I didn't know about that muriel could he used this way; this is great.
Do you know why it has this name?

Reply:
> Do you know why it has this name?

I found the Muriel project thread (p97Xot-wX-p2) from ages ago, and here's what it says:

> Why Muriel as a name?
> 
> We wanted to keep the name agnostic – aligning to best practices for naming Design Language Systems rather than to Automattic.